### PR TITLE
2 nc news

### PR DESCRIPTION
**Pull request by crossmatthew**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+describe('GET /api/topics', () => {
+    test('should return 200 OK status code and a body of all topics', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({ body }) => {
+            expect(Object.keys(body[0]))
+            .toEqual(expect.arrayContaining(
+                ['slug', 'description'])
+            )
```
Comment (MKD01) [inline __tests__/app.test.js]: This expect statement is specifically checking that the first topic has the keys "slug" and "description", but we should instead be expecting every topic to have these keys and ideally we should be expecting what the values of these are as well. 

I would recommend looking into .toMatchObject for this.

File: controllers/errorhandlers.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+exports.handleCustomErrors = (err, req, res, next) => {
+    if (err.status) {
+        res.status(err.status).send({message: err.msg})
+    }
+    next(err)
+}
```
Comment (MKD01) [inline controllers/errorhandlers.js]: This shouldnt be here yet. We should only be adding code that has been tested, which this cannot be tested for with the current endpoints you have.

File: __tests__/app.test.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    return db.end();
+});
+
+describe('GET /api/topics', () => {
+    test('should return 200 OK status code and a body of all topics', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({ body }) => {
+            expect(Object.keys(body[0]))
```
Comment (MKD01) [inline __tests__/app.test.js]: We should expect body to have a key of topics with the array of topics as its value.

**Review by MKD01 — CHANGES_REQUESTED** [review]
Good work so far, but I would like to see your tests updated before you merge this in, so please make the requested changes and re-submit this pr

**Comment by saimakhanom** [comment]
Well done for incorporating feedback - all good to merge!

**Pull request merged**